musicdroid/src/main/java/org/catrobat/musicdroid/pocketmusic/projectselection/ProjectSelectionActivity.java

```diff
@@ -120,6 +122,12 @@ public boolean onOptionsItemSelected(MenuItem item) {
             return true;
         }
 
+        if (id == R.id.action_about) {
+             DialogFragment newFragment = AboutDialog.newInstance(R.string.app_name);
+             newFragment.show(getFragmentManager(), "dialog");
+             return true;
```

Comment:
Why do you use a static newInstance Method?
Just do it f.e. like this: 

AboutDialog aboutDialog = new AboutDialog();
aboutDialog.show(getFragmentManager(), "dialog");